src/python/datastore/data_types.py

```diff
@@ -1294,6 +1294,12 @@ def _pre_put_hook(self):
                        coverage_information_key(self.fuzzer, self.date))
 
 
+class CorpusTag(Model):
+  """Corpus Tags for use in cross-pollination."""
```

Comment:
nit: I'd remove cross-pollination, as we can use this for other purposes to, maybe write something like "Corpus Tags for sharing corpora between fuzz targets."

Reply:
Done